themes/digital.gov/src/scss/new/_communities.scss

```diff
@@ -1,11 +1,29 @@
 @use "uswds-core" as *;
 
 // https://digital.gov/communities/
-.communities-of-practice--list {
-  @include at-media("tablet") {
-    column-gap: 2rem;
-    display: grid;
-    grid-template-columns: repeat(2, 1fr);
+
+.dg-communities {
+  @include u-flex("row", "wrap");
+  display: flex;
+  justify-content: center;
+  padding-top: units(2);
+
+  &-container-list {
+    margin: auto;
+    max-width: units(desktop);
+  }
+
+  &-title {
```

Comment:
Following BEM naming convention this would be more like `.dg-communities__title`.